src/libraries/Microsoft.Extensions.DependencyModel/src/DependencyContextLoader.cs

```diff
@@ -103,6 +105,7 @@ private DependencyContext LoadContext(IDependencyContextReader reader, string lo
             return null;
         }
 
+        [RequiresAssemblyFiles (Message = "Calls GetDepsJsonPath")]
```

Comment:
Also elsewhere. This is not the formatting we use in .NET. This is the formatting that illink repo uses just to upset people.

```suggestion
        [RequiresAssemblyFiles(Message = "Calls GetDepsJsonPath")]
```